src/core/vdrs/downloadVdrs.js

```diff
@@ -24,7 +24,7 @@ const downloadVdrs = async (vdrNames, jobId, processId, jobType, account) => {
         return null;
       }
       logDebug(`Downloading VDR for VDR name - ${vdrName}`, jobId);
-      const exportedVdr = await http.get(`/vdrs/${vdrName}/export`, {}, account);
+      const exportedVdr = await http.get(`vdrs/${vdrName}/export`, {}, account);
```

Comment:
@mukundbayyaram VDRs export is not working before? why do we need this change?

Reply:
@MohdRafiUddin 
The final URL of this API is resulting in `https://snapshot.cloud-elements.com/elements/api-v2//vdrs/accounts/export` with two `//`. Even though it is working, it's better to correct it and also can't have an incorrect test in doctor-service based on this.